### PR TITLE
validation phase workflow

### PR DESCRIPTION
**Pull request by vpchung**
- removed scattering steps from workflow
- kept the workflow input, `dataset`, but changed default value to `APOLLO-5`
- replaced `get_goldstandard_id` with a Synapse ID to the goldstandard
- updated `input_dir` from `leaderboard_data` to `final_input`

TODO (after doing some dry runs):
- do not return logs
- do not email score

### CODE REVIEW

File: workflow/docker_agent_workflow-final.cwl [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    type: string
+  - id: submitterUploadSynId
+    type: string
+  - id: workflowSynapseId
+    type: string
+  - id: synapseConfig
+    type: File
+  - id: dataset
+    type: string
+    default: ["APOLLO-5"]
```
Comment (thomasyu888) [inline workflow/docker_agent_workflow-final.cwl:29]: is this supposed to be a list?

Comment (vpchung) [reply workflow/docker_agent_workflow-final.cwl:29]: ah, good catch. updating now.

File: workflow/run_docker-final.py [hunk 2]
```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+"""Run training synthetic docker models"""
```
Comment (thomasyu888) [inline workflow/run_docker-final.py:2]:
```suggestion
"""Run final round models"""
```

File: workflow/docker_agent_workflow-final.cwl [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+        source: "#download_goldstandard/filepath"
+      - id: check_validation_finished
+        source: "#check_status/finished"
+      - id: dataset
+        source: "#dataset"
+    out:
+      - id: results
+
+  score_email:
+    run: https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v2.2/score_email.cwl
```
Comment (thomasyu888) [inline workflow/docker_agent_workflow-final.cwl:246]: Are we emailing scores back?

Comment (vpchung) [reply workflow/docker_agent_workflow-final.cwl:246]: oh i was keeping it for now while i do the dry runs. but no, we won't be.

**Pull request merged**